fadecandy_driver/src/fadecandy_driver.cpp

```diff
@@ -0,0 +1,301 @@
+#include "./fadecandy_driver.h"
+#include <cassert>
+#include <cstring>
+#include <iomanip>
+#include <iostream>
+#include <libusb-1.0/libusb.h>
+#include <list>
+#include <math.h>
+#include <vector>
+#define BITS 8
```

Comment:
Missing license header, alphabetic order and separate local includes from global includes